src/rtc.rs

```diff
@@ -46,6 +46,14 @@ pub enum Error {
     InvalidInputData,
 }
 
+/// Low Speed clock to be used by the RTC.
+pub enum LSClock {
+    /// External low speed clock, high-accuracy but requires external crystal
+    LSE,
+    /// Internal low speed clock, no crystal required but much lower accuracy
+    LSI,
```

Comment:
Nit: doc comment guidelines recommend a trailing full stop.

```suggestion
    /// External low speed clock. High-accuracy but requires external crystal.
    LSE,
    /// Internal low speed clock. No crystal required but much lower accuracy.
    LSI,
```